sdk/maps/azure-maps-search/README.md

```diff
@@ -0,0 +1,123 @@
+
+## Getting started
+
+## Prerequisites
+
+- Python 3.6 or later is required to use this package.
+- A deployed Maps Services resource. You can use the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp) or the [Azure PowerShell](https://docs.microsoft.com/powershell/module/az.communication/new-azcommunicationservice) to set it up.
+
+## Install the package
+
+Install the Azure Maps Service Search SDK.
+
+```bash
+pip install --pre azure-maps-search
+```
+
+## Authenticate the client
+
+In order to interact with the Document Translation feature service, you will need to create an instance of a client.
+An **endpoint** and **credential** are necessary to instantiate the client object.
+
+
+## Looking up the endpoint
+
+You can find the endpoint for your Maps resource using the
+[Azure Portal][azure_portal_get_endpoint].
+
+> Note that the service requires a custom domain endpoint. Follow the instructions in the above link to format your endpoint:
+> https://{NAME-OF-YOUR-RESOURCE}.mapsservices.azure.com/
+
+
+## Create the client with AzureKeyCredential
+
+To use an [API key] as the `credential` parameter,
+pass the key as a string into an instance of [AzureKeyCredential][azure-key-credential].
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.maps.search import SearchClient
+
+endpoint = "https://<resource-name>.mapsservices.azure.com/"
+credential = AzureKeyCredential("<api_key>")
+search_client = SearchClient(endpoint, credential)
+```
+
+## Create the client with an Azure Active Directory credential
+
+`AzureKeyCredential` authentication is used in the examples in this getting started guide, but you can also
+authenticate with Azure Active Directory using the [azure-identity][azure_identity] library.
+
+To use the [DefaultAzureCredential][default_azure_credential] type shown below, or other credential types provided
+with the Azure SDK, please install the `azure-identity` package:
+
+```pip install azure-identity```
+
+You will also need to [register a new AAD application and grant access][register_aad_app] to your
+Translator resource by assigning the `"Cognitive Services User"` role to your service principal.
+
+Once completed, set the values of the client ID, and client secret of the AAD application as environment variables:
+`AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`.
+
+```python
+from azure.identity import DefaultAzureCredential
+from azure.maps.search import SearchClient
+credential = DefaultAzureCredential()
+
+search_client = SearchClient(
+    endpoint="https://<resource-name>.mapsservices.azure.com/",
+    credential=credential
+)
+```
+
+## Key concepts
+
+to add all operations
+
+## Examples
+
+The following sections provide several code snippets covering some of the most common tasks, including:
+
+- [Operations](#events-operations)
```

Comment:
There's no `Operations` in this page. Please link real sample here. [Blob Storage](https://github.com/alextts627/azure-sdk-for-python/tree/maps-search-sdk-preview/sdk/storage/azure-storage-blob) or [KeyVault Keys](https://github.com/alextts627/azure-sdk-for-python/tree/maps-search-sdk-preview/sdk/keyvault/azure-keyvault-keys) are good examples to refer to.